lib/cnet/udp/udp_output.c

```diff
@@ -55,10 +55,7 @@ udp_output_header(pktmbuf_t *m, uint16_t nxt)
     udp->dgram_cksum = 0;
 
 #if CNET_ENABLE_IP6
```

Comment:
This ifdef can be removed as same condition has been added in is_pcb_dom_inet6().

Reply:
Removed the ifdef